README.md

```diff
@@ -23,11 +23,12 @@ creates the tags it needs on-the-fly in-memory without creating any files.
 * [Vim](http://www.vim.org/) >= 7.3.1058
   or any version of [NeoVim](https://neovim.io/).
 
-* [Exuberant Ctags](http://ctags.sourceforge.net/) >= 5.5,
-  or (**highly recommended**) any version of [Universal
-  Ctags](https://ctags.io) which is a currently maintained fork of Exuberant
-  Ctags with many bugfixes, support for many more formats, and proper Unicode
-  support. Some additional formats can also be handled by other providers such
+* A ctags implementation: We _highly recommend_ any version of [Universal
+  Ctags](https://ctags.io). It is a maintained fork of Exuberant Ctags with
+  many bugfixes, support for many more formats, and proper Unicode support.
+
+  Alternatively, use [Exuberant Ctags](http://ctags.sourceforge.net/) 5.5 or
+  higher. Some additional formats can also be handled by other providers such
```

Comment:
```suggestion
  Alternatively, use the [Exuberant Ctags](http://ctags.sourceforge.net/) 5.5 or
  higher. Note this supports less formats and will eventually be deprecated.
  
  Some additional formats can also be handled by other providers such
```

Reply:
Yes, makes sense. What about this version?